store/playbackManager.ts

```diff
@@ -201,9 +222,6 @@ export const mutations: MutationTree<PlaybackManagerState> = {
   ) {
     state.currentTime = time;
   },
-  RESET_CURRENT_TIME(state: PlaybackManagerState) {
-    state.currentTime = 0;
-  },
```

Comment:
Why remove this?

Reply:
@Camc314 Redundant, we commit time = 0 and it's the same thing. We can add it if we need it at some point, but I don't see an use case for this